components/PageHeader.vue

```diff
@@ -27,57 +27,35 @@
       v-else
     >
       <b-button
+        v-b-toggle.sidebar
         variant="light"
         class="navbar-toggle collapsed flex-column align-items-center justify-content-center align-self-center ml-3"
         :aria-label="$t('header.showSidebar')"
         data-qa="hamburger button"
-        @click="showSidebar = !showSidebar"
       >
         <span />
         <span />
         <span />
       </b-button>
-      <SmartLink
-        :destination="{ name: 'index' }"
-        class="logo d-inline-flex"
-      >
-        <img
-          src="../assets/img/logo.svg"
-          :alt="$t('homeLinkAlt')"
-          class="mw-100"
-          data-qa="logo"
-        >
-      </SmartLink>
-      <b-navbar
-        class="align-items-center flex-row d-flex p-0 mr-3"
-        role="navigation"
+      <b-sidebar
```

Comment:
running AXE with the sidebar open highlights a newly introduced issue.
Specifically no aria-label attribute.